megamek/src/megamek/client/ui/panels/phaseDisplay/lobby/LobbyMekPopup.java

```diff
@@ -1004,6 +1009,10 @@ private static boolean hasRapidFireMG(Entity entity) {
 
     /** Returns true when the entity has an MG set to normal (non-rapid) fire. */
     private static boolean hasNormalFireMG(Entity entity) {
+        // Battle armor cannot use burst-fire MGs per errata
```

Comment:
When wanting to be reeeeeal nitpicky, one might want to update the javadoc on the method by adding "*and* this entity is allowed to change it to rapid fire (i.e., not a BA)"